### DISCUSSION TITLE
concepts/hop/

### DISCUSSION THREAD

**Discussion by giscus** [go-gost/gost#385] [General]
# concepts/hop/

https://latest.gost.run/concepts/hop/

**Comment by pig0224** [comment]
转发器视乎无效

**Comment by yakamoz-fang** [comment]
跳跃点 http 接口返回值结构, 正确应该是: 
{
	"node": {
		"name": "http",
		"addr": ":8888",
		"connector": {
			"type": "http",
			"auth": {
				"username": "user",
				"password": "pass"
			}
		},
		"dialer": {
			"type": "tcp"
		}
	}
}

**Comment by Qiufen22** [comment]
热加载怎么无效，使用gost-ui，无法写入配置文件啊
